src/pipeline/pipeline.hpp

```diff
@@ -32,18 +32,13 @@ class Pipeline : public Producer<Product>, public Consumer<Consumable> {
     this->result = None();
 
     // Start pipeline.
+    // TODO(skonefal): Pipeline should not "return" with error.
     Try<Nothing> ret = this->produce(_product);
 
-    Try<Nothing> retPostRun = this->postPipelineRun();
```

Comment:
The same as before, but without faking that we have error handling :)

Seriously: 
1) Before, our components didn't check return value from _produce_, so we didn't had any errors returning back to Pipeline
2) We must move forward with the pipeline whenever we can
3) Currently we don't have any 'error' situations...
4) ... but we should create better scenario for it. Eg. some static "error" flag or carrying forward error messages